### PR TITLE
Add openapi spec and fix jsonschema generation for non crds

### PR DESCRIPTION
**Pull request by schrodit**
**How to categorize this PR?**

/area quality
/kind cleanup
/priority normal

**What this PR does / why we need it**:

Removed the previously used jsonschema generation library and now use the openapi spec generator of kubernetes.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
cc @enrico-kaack-comp 

**Release note**:

```bugfix user
The landscaper non-CRD resource jsonschemes are now generated using the Kubernetes OpenAPI-Library.
This fixes several issues with the current jsonschemas.
```

### CODE REVIEW

File: apis/.schemes/core-v1alpha1-Blueprint.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+          "default": ""
+        },
+        "required": {
+          "description": "Required specifies whether the import is required for the component to run. Defaults to true.",
+          "type": "boolean"
+        },
+        "schema": {
+          "description": "Schema defines the imported value as jsonschema.",
+          "type": "string",
+          "format": "byte"
```
Comment (enrico-kaack-comp) [inline apis/.schemes/core-v1alpha1-Blueprint.json:148]: incorrect schema definition. an import in the form
```
- name: ingressClass
  schema:
    type: string
```

should be allowed. (same for the export)

Maybe a problem resolving the  json.RawMessage

**Comment by schrodit** [comment]
@enrico-kaack-comp  PTAL

**Review by enrico-kaack-comp — APPROVED** [review]
one remark: in core-v1alpha1-Blueprint, the jsonSchemaVersion key seems to be defined as required. Not sure if this is wanted.

**Comment by schrodit** [comment]
> one remark: in core-v1alpha1-Blueprint, the jsonSchemaVersion key seems to be defined as required. Not sure if this is wanted.

the filed is required but defaulted, so I marked it as optional

**Review by enrico-kaack-comp — APPROVED** [review]
LGTM

**Pull request merged**